boards/riscv32/arty_bm310/doc/arty_bm310.rst

```diff
@@ -0,0 +1,65 @@
+.. _arty_bm310:
+
+CloudBEAR BM-310
+##############
+
+Overview
+********
+
+The CloudBEAR BM-310 is CloudBEAR IP design for Artix-7 35T Arty FPGA.
+More information about BM-310 can be found on
+`CloudBEAR's website <https://www.cloudbear.ru/bm_310.html>`_.
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``BM-310`` board configuration can be built as usual
+(see :ref:`build_an_application`).
+In order to build the application for ``BM-310``, set the ``BOARD`` variable
+to ``arty_bm310``.
+
+.. code-block:: bash
+
+   export BOARD="arty_bm310"
+
+Flashing
+========
+
+In order to upload the application to the device, you'll need OpenOCD
+and GDB with RISC-V support.
+Download and installation instructions can be found in the
+`RISC-V OpenOCD GitHub repository
+<https://github.com/riscv/riscv-openocd>`_ and
+`RISC-V GDB GitHub repository
+<https://github.com/riscv/riscv-binutils-gdb>`_ respectively.
+
+With the necessary tools installed, you can connect to the board using OpenOCD.
```

Comment:
Change to:

    With the necessary tools installed, establish an OpenOCD connection
    to the board using: